src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -473,6 +474,8 @@ export default class AirportModel {
             return;
         }
 
+        const airportPositionAndDeclination = [this.positionModel, this.magneticNorth];
```

Comment:
@n8rzz what do you think?

Reply:
not good, as I'm sure you can imagine.  Why not just create a getter for this?

```js
get airportPositionAndDeclination() {
    return [this.positionModel, this.magneticNorth];
}
```

Reply:
Doesn't the real issue lie with the way the parameters are being passed to getCircularCoordinates()? I don't see any reason to create and pass an array like this instead of passing the two parameters directly, eg

`getCircularCoordinates(area.center, area.radius, airportPosition, magneticDeclination);`

Reply:
Also, since this is not returning coordinates, but rather instances of `StaticPositionModel`, it ought to be renamed to reflect that. Perhaps `generatePositionsFromCircle()` or something.

Reply:
@erikquinn you know where my head is at there 😉 My fault for not looking very carefully.

I'd much prefer only passing what we need, that is certainly the better way to go.  And I agree on your second point as well.  Descriptive names are better names.